client/src/components/History/HistoryViews/JobHistoryModal.jsx

```diff
@@ -103,24 +107,32 @@ function JobHistoryContent({ jobDetails, jobDisplay, jobName, toggle }) {
     toggle();
   };
 
+  // TODO: This should be in jobDetails
   if ('queue' in jobDisplay) {
     configDataObj.Queue = jobDisplay.queue;
   }
 
-  configDataObj['Max Hours'] = jobDetails.maxHours;
+  // TODO: Convert to hours
+  configDataObj['Max Hours'] = jobDetails.maxMinutes;
 
+  // TODO: This should be in jobDetails
   if ('processorsPerNode' in jobDisplay) {
     configDataObj['Processors On Each Node'] = jobDisplay.processorsPerNode;
   }
+
+  // TODO: This should be in jobDetails
   if ('nodeCount' in jobDisplay) {
     configDataObj['Node Count'] = jobDisplay.nodeCount;
   }
+
+  // TODO: This should be in jobDetails
   if ('allocation' in jobDisplay) {
     configDataObj.Allocation = jobDisplay.allocation;
   }
 
   if (jobDetails.status !== 'FINISHED')
-    configDataObj['Temporary Working Directory'] = jobDetails.workPath;
+    // TODO: Switch to tapisv3
+    configDataObj['Temporary Working Directory'] = jobDisplay.workPath;
```

Comment:
You can use `execSystemExecDir` here in place of `workPath`. In fact, we want to show now:

- execSystemExecDir
- execSystemInputDir
- execSystemOutputDir